src/runner.ts

```diff
@@ -156,7 +156,7 @@ export class Runner {
       if (response !== null) {
         if (program.verbose) {
           const timing = (new Date().getTime() - this.timing) || 0;
-          if (response.messageTypes.includes(MessageType.Text)) {
+          if (response.attachments.length === 0) {
```

Comment:
This seems wrong? Should be `if (response.text)`. Also, for printing, we should probably print the attachments if they exist like:

```
Bot: Here you go: <IMAGE> <BUTTON> <BUTTON>
```

I know this is not how it is before but this would bring it into consistency with the new formatting style.

Reply:
Ah good catch, thanks.